core/pkg/artifacts/saver.go

```diff
@@ -761,16 +786,23 @@ func (as *ArtifactSaver) Save() (artifactID string, rerr error) {
 	} else if artifactAttrs.ArtifactSequence.LatestArtifact != nil {
 		baseArtifactId = &artifactAttrs.ArtifactSequence.LatestArtifact.Id
 	}
+
+	entity, project := as.fetchUseArtifactInputEntityAndProject()
+
 	if artifactAttrs.State == gql.ArtifactStateCommitted {
 		if as.artifact.UseAfterCommit {
-			_, err := gql.UseArtifact(
+			var err error
+			_, err = gql.UseArtifact(
 				as.ctx,
 				as.graphqlClient,
 				as.artifact.Entity,
 				as.artifact.Project,
 				as.artifact.RunId,
 				artifactID,
+				entity,
+				project,
 			)
+
```

Comment:
this code path is not covered, maybe it makes sense to add some tests to cover this?

Reply:
would love some ideas if there are easy ways to cover this. I tried introspecting the artifactSaver.save() code for useArtifact params and have not been successful (like in https://github.com/wandb/wandb/blob/main/core/pkg/artifacts/saver_test.go#L104)
Otherwise the saver tests and saver tests against the old server seems to be covering it a bit (it actually caught the fact the new parameters aren't generated properly).